editor/editor_paths.cpp

```diff
@@ -98,6 +98,18 @@ String EditorPaths::get_feature_profiles_dir() const {
 	return get_config_dir().path_join(feature_profiles_folder);
 }
 
+String EditorPaths::get_default_android_sdk_path() const {
+#ifdef WINDOWS_ENABLED
+	return OS::get_singleton()->get_environment("LOCALAPPDATA").path_join("Android/Sdk");
+#elif LINUXBSD_ENABLED
+	return OS::get_singleton()->get_environment("HOME").path_join("Android/Sdk");
```

Comment:
Could you clarify what installation process leads to the Android SDK being in this path on Linux?

In my case I install the Android SDK from the command line tools from Google and thus the path is whatever I unzip the tools at, and pass as `--sdk_root` to `sdkmanager`.

Is `~/Android/Sdk` the default path used by Android Studio on Linux?

Reply:
Yes, in my case Android Studio (no matter if it's snap, flatpak or official version) always use '~/Android' location for its files, including the SDK.